tools/health/monitorUtil.py

```diff
@@ -46,8 +46,9 @@ def run(cmd, inData="") :
         p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stdin=subprocess.PIPE, stderr=subprocess.STDOUT)
         if (inData != ""):
             p.stdin.write(inData)
-        p.wait()
```

Comment:
This file is being used by `tools/health/isAlive`.
And `tools/health/isAlive` is being used by `services.PingTests`